iCubGenova09/hardware/motorControl/right_leg-eb11-j2_3-mc_service.xml

```diff
@@ -47,7 +47,7 @@
                     <param name="type">                 roie            roie       </param>
                     <param name="port">                 CAN1:1:0        CAN1:2:0   </param>
                     <param name="position">             atmotor         atmotor    </param>
-                    <param name="resolution">           14400           3600      </param>
+                    <param name="resolution">           14400           14400      </param>
```

Comment:
Already in `devel`?!
See https://github.com/robotology/robots-configuration/blob/devel/iCubGenova09/hardware/motorControl/right_leg-eb11-j2_3-mc_service.xml#L50.